reports/Alabama.md

```diff
@@ -1,3 +1,18 @@
+## Hoover
+
+### Police make a series of violent arrests | Date Unknown 
+
+During a peaceful protest, police corral protestors onto a hill. Officers snatch protestors at the edges and throw them to the ground to make arrests.
+
+tags: arrest, shove, throw, push, knee
+
+id: al-hoover-1
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1269810926863163394
```

Comment:
I searched for the original tweet Doucette's referencing here and found it. Could be good to use that as the original source?

```suggestion
* https://twitter.com/mtvtacallya/status/1267318555604910081
```

Reply:
Yeah thanks. I usually try to find Doucette's source but struggled this time